### ISSUE TITLE
Bug: Style Tweaks to Repo Page

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#3351]
Nothing showstopper here, just a bit of cleanup.

![CleanShot 2024-05-09 at 08 47 26](https://github.com/open-sauced/app/assets/833231/d35b681a-ddde-4b7f-9291-de14aa8d4ba4)

1. Add some padding between the last graphs and the footer.
2. The graphs on some screen sizes get misaligned for the legend.
3. I also wonder if on smaller screens we just put PRs instead of pull requests. Thoughts @isabensusan?

**Comment by isabensusan** [comment]
yeah, there's some polish to be done, but probably easier / faster if we pair on it @zeucapua
